catalog/app/containers/Bucket/BucketNav.tsx

```diff
@@ -79,6 +79,9 @@ function Tabs({ bucket, preferences, section = false }: TabsProps) {
       {section === 'search' && (
         <NavTab label="Search" value="search" to={urls.bucketSearch(bucket)} />
       )}
+      {(section === 'queries' || section === 'es') && (
```

Comment:
this condition should be probably changed to `{preferences.queries && (`, since otherwise ES tab is only shown while it's already active (or while the queries tab is active)

Reply:
RIght, thanks!

Reply:
Oh, I didn't get your point.

The goal was to show ES tab only when ES or Queries tab is active. Similar to Search tab

Reply:
I added `preferences.queries` condition but in addition to existing condition

Reply:
well ok if that's intentional, i just i though it makes little sense showing extra ES tab after switching to Queries tab

Reply:
I'd better move ES link to somewhere on Queries tab, but didn't find a proper place for it.

Maybe, we can move it to dropdown help in search bar :thinking: